crates/rnote-ui/src/appwindow/mod.rs

```diff
@@ -104,6 +104,11 @@ impl RnAppWindow {
         self.imp().overlays.get()
     }
 
+    #[allow(unused)]
```

Comment:
Really minor nit, but I always add getter *and* setter's for gobject properties. (And move it slightly higher, below the other getters/setters in L86)